docs/rst/manual/about/release_notes.rst

```diff
@@ -1,6 +1,75 @@
 Release Notes
 =============
 
+Version 2.10
+------------
+
+Highlighted changes
+~~~~~~~~~~~~~~~~~~~
+
+The shell script jobs will no longer interpret the first path relative to the
+configuration file. This implies that if you want to copy a file from the area
+around your configuration file to the runpath, the following copying will not
+work anymore:
+
+.. code-block::
+
+   FORWARD_MODEL COPY_FILE(<FROM>=my_files/data.txt, <TO>=data.txt)
+
+And the reason is that it is not possible to deduce whether the intent was to
+copy a file relative to your configuration file or whether you wanted to copy
+(or delete) something that is already in your runpath. This led to mysterious
+and strange errors. From now on, you will have to explicitly ask for the path
+to be relative to your configuration file using the ``<CONFIG_PATH>`` magic
+string:
+
+.. code-block::
+
+   FORWARD_MODEL COPY_FILE(<FROM>=<CONFIG_PATH>/my_files/data.txt, <TO>=data.txt)
+
+The above change takes effect for the following shell scripts:
+  - ``CAREFUL_COPY_FILE``
+  - ``COPY_DIRECTORY``
+  - ``COPY_FILE``
+  - ``DELETE_DIRECTORY``
+  - ``DELETE_FILE``
+  - ``MAKE_DIRECTORY``
+  - ``MAKE_SYMLINK``
+  - ``MOVE_FILE``
+  - ``SCRIPT``
+  - ``SYMLINK``
+
+2.10 ERT application
+~~~~~~~~~~~~~~~~~~~~
+
+Improvements:
+  - Improve documentation on ARGSLIST
+  - Enable RML_ENKF in default ERT installation
+
+Bug fixes:
+  - Fixed jumping cursor when filling in ES MDA weights
+  - Fixed Python exception on exit
+  - Fixed how logarithmic plots handles negative values
+  - Do not terminate on exception in RunContext due to race condition
```

Comment:
This isn't true just yet: https://github.com/equinor/ert/compare/2.10.0...master

Should we create a 2.10.1 tag?

Reply:
Yeah, will do immediately 👍